src/panels/config/automation/action/types/ha-automation-action-choose.ts

```diff
@@ -64,12 +65,31 @@ export class HaChooseAction extends LitElement implements ActionElement {
   @consume({ context: fullEntitiesContext, subscribe: true })
   _entityReg!: EntityRegistryEntry[];
 
-  @state()
-  @consume({ context: reorderModeContext, subscribe: true })
-  private _reorderMode?: ReorderMode;
+  @state() private _showReorder: boolean = false;
 
   private _expandLast = false;
 
+  private _mqlListenerRef?: (event: MediaQueryListEvent) => void;
+
+  private _mql?: MediaQueryList;
+
+  public connectedCallback() {
+    super.connectedCallback();
+    this._mql = window.matchMedia("(min-width: 600px)");
```

Comment:
Does this have to be `600px`, can we maybe use `narrow`?

https://github.com/home-assistant/frontend/blob/dev/src/layouts/home-assistant-main.ts#L54

Reply:
Narrow is bigger (870px) and we have to pass it to every component so I preferred to use it like this.